app/src/main/java/app/gamenative/service/SteamService.kt

```diff
@@ -2480,6 +2493,30 @@ class SteamService : Service(), IChallengeUrlChanged {
             PluviaApp.events.emit(event)
         } ?: run { Timber.w("QR challenge url was null") }
     }
+
+    /**
+     * Fetches owned games data and caches playtime values for use in PICS processing.
+     * This is called once per login session in onLicenseList.
+     */
+    private fun fetchAndCachePlaytimeData() {
```

Comment:
Using the getOwnedGames() this allows us to grab game information and specifically playtime. 

We then persist this to a cache for storing steamApps()